delivery_free_fee_removal/models/sale_order_line.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2020 Camptocamp
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class SaleOrderLine(models.Model):
+    _inherit = "sale.order.line"
+
+    is_free_delivery = fields.Boolean(compute="_compute_is_free_delivery", store=True,)
+
+    @api.depends("is_delivery", "currency_id", "price_unit")
+    def _compute_is_free_delivery(self):
+        for line in self:
+            line.is_free_delivery = line.is_delivery and line.currency_id.is_zero(
+                line.price_unit
```

Comment:
Maybe `line.price_reduce`  for taking into account 100% discount?

Reply:
Much better yes! I'll update my PR

Reply:
Actually `price_total` should cover all the possible cases.